opslevel/resource_opslevel_domain.go

```diff
@@ -123,6 +122,7 @@ func (r *DomainResource) Create(ctx context.Context, req resource.CreateRequest,
 	}
 	createdDomainResourceModel, diags := NewDomainResourceModel(ctx, *resource)
 	resp.Diagnostics.Append(diags...)
+	createdDomainResourceModel.LastUpdated = types.StringValue(timeLastUpdated())
```

Comment:
This might be a hold over from the TF SDK v1 - we should see if this field makes sense in the new world.